src/game/delete-round.ts

```diff
@@ -14,25 +14,96 @@ export class DeleteRound extends Round {
         super();
     }
 
-    private async getColumnOffset(game: IGame) {
-        const isDefindedUserOffset = await game.nvim.eval(
-            'exists("vim_be_good_delete_me_offset")',
+    private async getGameOptions(game: IGame) {
+        type userOptionsType = { [key: string]: number };
+
+        let randomOffset: number = 0;
+        let fixedOffset: number = 0;
+
+        const isDefinedUserRandomOffset = await game.nvim.eval(
+            'exists("vim_be_good_delete_me_fixed_offset")',
```

Comment:
I debugged this for like an hour, completely confused... Then I realized that lines 24 and 27 are backward!

🤦

Reply:
omg, sorry @brandoncc ;)

Reply:
can't believe I also didn't see that! ahhahaha